middleware/src/rpcmessages/errorcodes.go

```diff
@@ -136,9 +138,31 @@ const (
 	// ErrorDummyAuthenticationNotSuccessful is thrown if the dummy authentication is not successful.
 	ErrorDummyAuthenticationNotSuccessful ErrorCode = "DUMMY_AUTHENTICATION_NOT_SUCCESSFUL"
```

Comment:
Originally dummy was meant in the way: "This is a placeholder once we have JWT Auth." Is it still needed? Could be it the case that you are using it for the "ICanHasPassword?" setup Auth?

Reply:
Yes, I removed the dummy prefix and renamed it to 'initial'.